src/tiles/Tiles.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState } from "react";
+import {
+  TileName,
+  IconContainer,
+  FlippedTile,
+  TileContent,
+  Tile,
+} from "./TileStyles";
+import { ThemeContext } from "../ThemeContext";
+import { getRandomSize } from "../utils";
+
+export const Tiles = ({
+  tileName,
+  content,
+  isSquare,
+  color,
+  shuffleColors,
+}) => {
+  const isDarkMode = useContext(ThemeContext);
+  const [isFlipped, setIsFlipped] = useState(false);
+
+  const tileStyle = {
```

Comment:
I think this can be moves to the styles file.

Reply:
The reason being?

Reply:
Because it basically sets the tike color which is a css attribute.